src/components/core/FontStyle.js

```diff
@@ -82,10 +59,12 @@ const FontStyle = ({
         {onColorChange && (
             <ColorPicker
                 color={cssColor(color) || LABEL_FONT_COLOR}
-                width={itemSize}
-                height={itemSize}
+                width={32}
+                height={32}
                 onChange={onColorChange}
-                style={styles.color}
+                style={{
+                    margin: '12px 8px',
```

Comment:
why not put this in `FontStyle.module.css`?

Reply:
Main reason is that all components are not translated to @dhis2/ui yet, and css modules can't be used with style prop as it returns class names. But now I've started - just a warning that this PR will big!

Reply:
Fixed.